ui/shared/components/panel/variants/VariantClassify.jsx

```diff
@@ -0,0 +1,115 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { Button, Label } from 'semantic-ui-react'
+import { CopyToClipboard } from 'react-copy-to-clipboard'
+import { HttpRequestHelper } from 'shared/utils/httpRequestHelper'
+import { ColoredIcon } from 'shared/components/StyledComponents'
+import DataLoader from 'shared/components/DataLoader'
+import AcmgModal from '../acmg/AcmgModal'
+import { VerticalSpacer } from '../../Spacers'
+import PopupWithModal from '../../PopupWithModal'
+import { getVariantMainTranscript } from '../../../utils/constants'
+
+const CLINGEN_ALLELE_REGISTRY_URL = 'http://reg.genome.network/allele'
+const CLINGEN_VCI_URL = 'https://curation.clinicalgenome.org/select-variant'
+
+class ClinGenAlleleId extends React.PureComponent {
+
+  static propTypes = {
+    hgvsc: PropTypes.string.isRequired,
+  }
+
+  state = {
+    copied: false,
+    loading: false,
+    alleleId: null,
+  };
+
+  load = (hgvsc) => {
+    this.setState({ loading: true })
+    new HttpRequestHelper(CLINGEN_ALLELE_REGISTRY_URL,
+      (responseJson) => {
+        this.setState({ alleleId: responseJson['@id'].split('/').pop(), loading: false })
+      },
+      (e) => {
+        this.setState({ alleleId: e.message, loading: false })
```

Comment:
can I see a screenshot of how it looks in this case?

Reply:
Sure, it is as below figure. Maybe we should hide the copy button in case of an error.
![image](https://user-images.githubusercontent.com/33550059/218545526-d6247097-9c32-4e2b-974f-a988b25e5ded.png)